install-sdk/package-lock.json

```diff
@@ -1249,8 +1249,8 @@
       }
     },
     "@types/node": {
-      "version": "12.12.14",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-12.12.14.tgz",
+      "version": "16.10.0",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-16.10.0.tgz",
       "integrity": "sha512-u/SJDyXwuihpwjXy7hOOghagLEV1KdAST6syfnOk6QZAMzZuWZqXy5aYYZbh8Jdpd4escVFP0MvftHNDb9pruA==",
```

Comment:
@Malinskiy I've no clue what the right `integrity` value is and how to calculate it